libsel4/sel4_arch_include/aarch64/sel4/sel4_arch/deprecated.h

```diff
@@ -28,6 +28,10 @@
 #define seL4_PUDBits 13
 #define seL4_PUDIndexBits 10
 #define seL4_ARM_PageUpperDirectoryObject seL4_ARM_VSpaceObject
+#define seL4_ARM_PageUpperDirectory seL4_ARM_VSpace
+
+/* this was previously defined always, so keep that here for consistency */
+#define seL4_ARM_PageGlobalDirectory seL4_CPtr
```

Comment:
This can be `seL4_ARM_PageTable`. But as it all ends up the same, there's no type checking anyway and it doesn't really matter.